runhouse/utils.py

```diff
@@ -800,3 +810,335 @@ def print_sky_clusters_msg(num_sky_clusters: int):
+                float(used_gpu_memory / total_gpu_memory) * 100, 2
+            )
+            gpu_usage_summery = f"{DOUBLE_SPACE_UNICODE}GPU Memory: {used_gpu_memory} / {total_gpu_memory} Gb ({gpu_memory_usage_percent}%)"
+            console.print(gpu_usage_summery)
+
+        resources_in_env = [
+            {resource: resources_in_env[resource]}
+            for resource in resources_in_env
+            if resource is not env_name
+        ]
+
+        if len(resources_in_env) == 0:
+            # No resources were found in the env, only the associated installed python reqs were installed.
+            console.print(
+                f"{DOUBLE_SPACE_UNICODE}This environment has only python packages installed, if provided. No resources were "
+                "found."
+            )
+
+        else:
+            for resource in resources_in_env:
+                for resource_name, resource_info in resource.items():
+                    resource_type = adjust_resource_type(
+                        resource_info.get("resource_type")
+                    )
+
+                    active_function_calls = resource_info.get("active_function_calls")
+                    resource_info_str = f"{DOUBLE_SPACE_UNICODE}{BULLET_UNICODE} {resource_name} ({resource_type})"
+
+                    if resource_type == "runhouse.Function" and active_function_calls:
+                        func_start_time_utc = active_function_calls[0].get(
+                            "start_time", None
+                        )
+
+                        # casting func_start_time_utc to datetime format
+                        func_start_time_utc = datetime.fromtimestamp(
+                            func_start_time_utc, tz=timezone.utc
+                        )
+
+                        # func_end_time_utc = current time. Making sure it is in the same format as func_start_time_utc,
+                        # so we could calculate function's running time.
+                        func_end_time_utc = datetime.fromtimestamp(
+                            time.time(), tz=timezone.utc
+                        )
+
+                        func_running_time = (
+                            func_end_time_utc - func_start_time_utc
+                        ).total_seconds()
+
+                        is_func_running: str = f" [italic bright_green]Running for {func_running_time} seconds[/italic bright_green]"
+
+                    elif (
+                        resource_type == "runhouse.Function"
+                        and not active_function_calls
+                    ):
+                        is_func_running: str = " [italic bright_yellow]Currently not running[/italic bright_yellow]"
+
+                    else:
+                        is_func_running: str = ""
+
+                    resource_info_str = resource_info_str + is_func_running
+
+                    console.print(resource_info_str)
+
+
+def print_cloud_properties(cluster_config: dict):
```

Comment:
caroline recently added a few more properties (https://github.com/run-house/runhouse/pull/1246/files), can we add these too?

Reply:
current output:
![image](https://github.com/user-attachments/assets/fc5a1028-512b-4be0-93c1-93946b5ec83f)

printing the `num_cpus` and accelerators info, if exists. Not sure if we need to print the total disk size. @jlewitt1 wdyt?

Reply:
yea i think thats fine for now